src/lxc/start.c

```diff
@@ -97,14 +97,16 @@ static void lxc_destroy_container_on_signal(struct lxc_handler *handler,
 
 static void print_top_failing_dir(const char *path)
 {
+	__attribute__((__cleanup__(__auto_free__))) char *copy;
 	int ret;
 	size_t len;
-	char *copy, *e, *p, saved;
+	char *e, *p, saved;
 
 	len = strlen(path);
-	copy = alloca(len + 1);
+	copy = must_realloc(NULL, len + 1);
 	(void)strlcpy(copy, path, len + 1);
```

Comment:
The above 2 lines should be removed with the `must_copy_string()` around below.

Reply:
done